aspnetcore/grpc/test-tools.md

```diff
@@ -50,6 +50,30 @@ When gRPC reflection is set up:
 * Client apps that support gRPC reflection can call the reflection service to discover services hosted by the server.
 * gRPC services are still called from the client. Reflection only enables service discovery and doesn't bypass server-side security. Endpoints protected by [authentication and authorization](xref:grpc/authn-and-authz) require the caller to pass credentials for the endpoint to be called successfully.
 
+### Reflection service security
+
+gRPC reflection returns a list of available APIs, which could be sensitive information. Care should be taken to limit access to the gRPC reflection service.
+
+gRPC reflection is often only required in a local development environment. In this situation, the service should only be mapped when <xref:Microsoft.AspNetCore.Hosting.HostingEnvironmentExtensions.IsDevelopment(IHostingEnvironment)> returns true:
```

Comment:
```suggestion
gRPC reflection is often only required in a local development environment. In this situation, the service should only be mapped when [IsDevelopment](/en-us/dotnet/api/microsoft.aspnetcore.hosting.hostingenvironmentextensions.isdevelopment) returns true:
```

Reply:
Looks like Rick and I are in at same time, I will get out.